transform/tcga/gistic2_cna.py

```diff
@@ -0,0 +1,104 @@
+from collections import defaultdict
+import csv
+from glob import glob
+import os.path
+import gzip
+import json
+import logging
+import subprocess
+import sys
+
+import bmeg.enrichers.gene_enricher as gene_enricher
+from bmeg.vertex import Aliquot, CopyNumberAlteration
+from bmeg.edge import CopyNumberAlterationOf
+from bmeg.emitter import JSONEmitter
+from bmeg.util.cli import default_argument_parser
+from bmeg.util.logging import default_logging
+
+
+def transform(source_path,
+              aliquot_path="outputs/gdc/Aliquot.Vertex.json.gz",
+              emitter_directory="tcga",
+              source="tcga",
+              method="gistic2"):
+
+    # check if we are doing one file at time
+    p, file_name = os.path.split(source_path)
+    prefix = file_name.split('_')[0]
+    emitter = JSONEmitter(directory=emitter_directory, prefix=prefix)
+    logging.debug('individual file prefix {}'.format(prefix))
+
+    aliquot_lookup = {}
+    with gzip.open(aliquot_path) as handle:
+        for line in handle:
+            a = json.loads(line)
+            barcode = a["data"]["gdc_attributes"]["submitter_id"]
+            gdc_id = a["data"]["gdc_attributes"]["aliquot_id"]
+            aliquot_lookup[barcode] = gdc_id
+
+    reader = csv.reader(open(source_path, "rt"), delimiter="\t")
+    header = next(reader)
+    samples = header[3:]
+
+    # collect expression for all aliquots and transcripts
+    collect = defaultdict(dict)
+
+    for row in reader:
+        feature_ids = row[0].split("|")
+        symbol = feature_ids[0]
+        try:
+            genes = gene_enricher.get_gene(symbol)
+            ensembl_id = genes[0]['ensembl_gene_id']
+            symbol = ensembl_id
+        except Exception as e:
+            logging.warning(str(e))
+            continue  # do we want to drop these data?
```

Comment:
Right now I am stripping out the data that isn't able to be mapped to ensembl ids.. @kellrott would you prefer I leave these in?